app/containers/Vaults/saga.js

```diff
@@ -407,6 +432,17 @@ function* migrateVault(action) {
         trustedMigratorContract.address,
       );
     }
+
+    yield call(
+      trustedMigratorContract.methods.migrateAll.cacheSend,
+      vaultMigrationData.vaultFrom,
+      vaultMigrationData.vaultTo,
+      {
+        from: account,
+        gas: 2800000,
+      },
+    );
+
     yield call(
       trustedMigratorContract.methods.migrateAll.cacheSend,
```

Comment:
should we delete one call? It will try to execute one after the other

Reply:
yeah this appears to be setting all vaults to 2800000 gas limit...which should be fine, but means we don't need the second call and should move the error catch up to the first one probably, right?

Reply:
resolved